bentoml/_internal/bento/bento.py

```diff
@@ -71,6 +71,7 @@ def create(
 
             model_name, model_version = model_tag.split(":")
             target_path = os.path.join("models", model_name, model_version)
+            bento_fs.makedirs(target_path, recreate=True)
```

Comment:
can we create a tmp repo path?

Reply:
i think bento_fs is already a tmp path `bento_fs = fs.open_fs(f"temp://bentoml_bento_{svc.name}")`